hack/update-codegen.sh

```diff
@@ -51,6 +51,7 @@ ${GOPATH}/bin/deepcopy-gen \
   -i github.com/knative/serving/pkg/reconciler/v1alpha1/revision/config \
   -i github.com/knative/serving/pkg/reconciler/v1alpha1/route/config \
   -i github.com/knative/serving/pkg/autoscaler \
```

Comment:
Do we still need this?

Reply:
Yes, for copying the `Metric` type defined in `multiscaler.go`.